lib/ProductOpener/Products.pm

```diff
@@ -210,6 +268,17 @@ sub init_product($) {
 		rev=>0,
 	};
 
+	if ((defined $server_options{private_products}) and ($server_options{private_products})) {
+		my $owner = "user-" . $userid;
```

Comment:
Potentially conflicts with `$ProductOpener::Display::owner`.

Reply:
I think I can remove Display.pm being loaded in Products.pm, which would make sure $owner does not come from Display.pm

Reply:
Would be a good step towards #622. :)

Reply:
Would be a good step towards #622. :)